common/extensions/api/brave_rewards.json

```diff
@@ -369,6 +369,172 @@
         "description": "Creates new wallet",
         "parameters": []
       },
+      {
+        "name": "updateMediaDuration",
+        "type": "function",
+        "description": "Update the media duration for a given resource",
+        "parameters": [
+          {
+            "name": "publisherKey",
+            "type": "string"
+          },
+          {
+            "name": "duration",
+            "type": "integer"
+          }
+        ]
+      },
+      {
+        "name": "getPublisherInfo",
+        "type": "function",
+        "description": "Get the details for a particular publisher",
+        "parameters": [
+          {
+            "name": "publisherKey",
+            "type": "string"
+          },
+          {
+            "type": "function",
+            "name": "callback",
+            "parameters": [
+              {
+                "name": "result",
+                "type": "integer"
+              },
+              {
+                "name": "properties",
+                "type": "object",
+                "properties": {
+                  "publisher_key": {
```

Comment:
please use camel case as this is JavaScript so `publisherKey`

Reply:
I think it ended up this way because `rewardsExtensions.d.ts` defines it as `publisher_key` in the `Publisher` type. I'll change it there and update these.